frontend/src/metabase/query_builder/components/DatasetEditor/DatasetEditor.jsx

```diff
@@ -480,16 +472,11 @@ function DatasetEditor(props) {
         data-testid="dataset-edit-bar"
         title={question.displayName()}
         center={
-          // Metadata tab is temporarily disabled for metrics.
-          // It should be enabled in #37993
-          // @see https://github.com/metabase/metabase/issues/37993
```

Comment:
This comment is no longer true. In the current form where metrics are just aggregations, they would likely not have metadata at all or it would be very different to what we have with models.